packages/notifier/README.md

```diff
@@ -51,10 +62,14 @@ publication.updateState('a');
 publication.updateState('b');
 publication.finish('done');
 ```
-The publisher has published an interation with the non-final sequence
-`'a'`, `'b'`, terminated with `'done'` as its completion value.
 
-Alice the subscriber consumes using the `for-of-await` loop
+Alice, the subscriber, consumes the iteration using the `for-of-await` loop. She can see the 
+non-final values and whether the iteration completes or fails. She can see a failure reason, 
+but the `for-await-of` syntax does not let her see the completion value `'done'`. While she 
+can write code that only executes after the loop finishes, the code won’t know if the completion 
+value was “done”, “finished”, or something else. This is a limitation of JavaScript's iteration, 
```

Comment:
Choose an example string other than `'finished'` here to illustrate the point, since Alice's code emits `'finished'` to indicate that it detected successful termination.